channeldb/meta.go

```diff
@@ -0,0 +1,78 @@
+package channeldb
+
+import (
+	"github.com/boltdb/bolt"
+)
+
+var (
+	// metaBucket stores all the meta information concerning the state of
+	// the database.
+	metaBucket = []byte("metadata")
+
+	// dbVersionKey is a boltdb key and it's used for storing/retrieveing
+	// current database version.
+	dbVersionKey = []byte("dbp")
+)
+
+// Meta structure holds the database meta information.
+type Meta struct {
+	dbVersionNumber uint32
+}
+
+func (d *DB) FetchMeta() (*Meta, error) {
+	meta := &Meta{}
+	err := d.store.View(func(tx *bolt.Tx) error {
+		metaBucket := tx.Bucket(metaBucket)
+		if metaBucket == nil {
+			return ErrMetaNotFound
+		}
+
+		fetchDbVersion(metaBucket, meta)
+
+		return nil
+	})
+
+	if err != nil {
```

Comment:
Minor nit: to save a few lines, the `meta` variable and `err` can just be returned directly. 

So: 
```go
return meta, err
```